compiler/next/lib/parsing/ParserContextImpl.h

```diff
@@ -32,6 +32,8 @@
 
 using chpl::types::Param;
 
+void gdbShouldBreakHere(void);
```

Comment:
Is this related to the other changes in this file?

Reply:
Oops, nope, I was using that to debug some stuff in the parser. Thanks!